docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/{{ cookiecutter.module_name }}/management/commands/load_cms_content.py

```diff
@@ -0,0 +1,93 @@
+# Based on the Wagtail demo:
+# https://github.com/wagtail/bakerydemo/blob/master/bakerydemo/base/management/commands/load_initial_data.py
+import os
+
+from django.core.files.storage import default_storage
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from django.core.exceptions import ObjectDoesNotExist
+
+from wagtail.core.models import Site, Page, PageRevision
+from wagtail.images.models import Image
+
+
+class Command(BaseCommand):
+
+    @property
+    def fixtures_dir(self):
+        project_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+        return os.path.join(project_dir, 'fixtures')
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--content-type',
+            default='models',
+            type=lambda x: x.split(','),
+            help='Comma-separated string of content types to load. Default: \
+                  models. Options: models, images.'
+        )
+
+    def handle(self, **options):
+        selected_content = set(options['content_type'])
+        valid_content = set(['models', 'images'])
+
+        if not selected_content.issubset(valid_content):
+            invalid_values = selected_content - valid_content
+            raise ValueError('Invalid content types: {}. Valid options are: \
+                              models, images'.format(invalid_values))
+
+        else:
+            for content in options['content_type']:
+                getattr(self, 'load_{}'.format(content))()
+
+    def load_models(self):
+        initial_data_file = os.path.join(self.fixtures_dir, 'initial_cms_content.json')
+        initial_data_file_custom_pages = \
+            os.path.join(self.fixtures_dir,
+                         'initial_cms_content_custom_pages.json')
+
+        # Delete existing Wagtail models
+        try:
+            Site.objects.all().delete()
+            Page.objects.all().delete()
+            PageRevision.objects.all().delete()
+            Image.objects.all().delete()
```

Comment:
If we do want to flush existing content, how about checking whether there are pages in the fixture? If not, exit the command. Otherwise, we're safe to proceed with flushing and loading.

Reply:
Since `load_cms_content` will be used anytime we can to update the default content, we do want to flush existing content before loading a Django dump. I think checking if `initial_cms_content.json` is empty is a great idea.